bin/concordance_validation/validate_and_delta.rb

```diff
@@ -25,9 +25,9 @@ def review_logs(log_file)
 
 conc_dir = Settings.concordance_path
 
-raw_gzip_files = Dir.glob("#{conc_dir}/raw/*txt.gz").sort
+raw_files = Dir.glob("#{conc_dir}/raw/*txt*").sort
 
-file_names_split = raw_gzip_files.map {|fname| fname.split("/").last.split("_").first }
+file_names_split = raw_files.map {|fname| fname.split("/").last.split("_").first }
 raw_dates = file_names_split.select {|d| d =~ /^\d+$/ }
```

Comment:
Might be more clear here to use a regex here -- if nothing else, it would be worth a comment here as to what filename format this is expecting.

Reply:
Commented expected file format.

Reply:
And now with regex instead of split("_").first